smoketest/tests/send_tokens.rs

```diff
@@ -71,19 +64,10 @@ async fn send_tokens() {
         .unwrap();
     assert_eq!(receipt.status.unwrap().as_u64(), 1u64);
 
-    let recipient = MultiLocation {
-        parents: 0,
-        interior: X1(Junction::AccountId32 {
-            network: None,
-            id: FERDIE,
-        }),
-    }
-    .encode();
-
     // Lock tokens into vault
     let value1: u128 = U256::from(value).low_u128();
-    native_tokens
-        .lock(weth.address(), 0, recipient.into(), value1)
+    gateway
+        .send_token(weth.address(), 1000.into(), FERDIE, value1)
         .value(1000)
         .send()
         .await
```

Comment:
Would suggest we log the transaction hash here.

Reply:
Resolved in [0c4bdcc](https://github.com/Snowfork/snowbridge/pull/904/commits/0c4bdccbe776bfeb53b2d24c4437b4a0c74b6fa2)